airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -29,6 +32,16 @@ class GoogleCloudBaseHook(BaseHook):
     for a Google cloud service.
 
     The class also contains some miscellaneous helper functions.
+
+    All hook derived from this base hook use the 'Google Cloud Platform' connection
+    type. Two ways of authentication are supported:
+
+    Default credentials: Only specify 'Project Id'. Then you need to have executed
```

Comment:
Does scope need to be supplied when using default credentials? I haven't tried this.